src/schema/v2/types/Video.ts

```diff
@@ -8,28 +8,37 @@ import {
 import uuid from "uuid/v5"
 import { ResolverContext } from "types/graphql"
 
-export const VideoType = new GraphQLObjectType<any, ResolverContext>({
-  name: "Video",
-  description: "An object containing video metadata",
-  fields: {
-    id: {
-      type: new GraphQLNonNull(GraphQLID),
-      resolve: ({ url }) => {
-        return uuid(url, uuid.URL)
+interface VideoTypeProps {
+  id: string
+  playerUrl: string
+  height: number
+  width: number
+}
```

Comment:
It'd be good to annotate the return of the function that constructs this object with this type. (Or import the `ReturnType` of that function here instead.)

Reply:
Not sure that I follow? Like, the return type of `uuid`? (Can follow-up)

Reply:
`getFigures` returns: ![](https://static.damonzucconi.com/_capture/E1V18uZ4.png)

Reply:
so make that like: `NormalizedImageData | VideoData` and then import `VideoData` here.

Reply:
Oh I see! Will do 👍